app/helpers/application_formatters_helper.rb

```diff
@@ -94,9 +94,9 @@ def display_topic(topic, options = {})
   #        :date_only_long => "December 04, 2007"
   #        :date_only_short => "04 Dec"
   # @return [String] the formatted datetime string
-  def format_datetime(date, format = :long)
-    user_zone = (current_user ? current_user.time_zone : nil) ||
-                Application.config.x.default_user_time_zone
+  def format_datetime(date, format = :long, user = nil)
```

Comment:
Prefer to use keyword arguments if arg list is too long. `format_datetime(date, format: :long, user:)`

Reply:
This method has already been used in too many places, we need to change a lot, if we make the ```format``` as a key word argument. I would prefer only make the user as a keyword argument?

Reply:
OK, sure